.github/workflows/pull-request-account-compression.yml

```
@@ -0,0 +1,90 @@
+name: Account Compression Pull Request
+
+on:
+  pull_request:
+    paths:
+      - "account-compression/**"
+      - "libraries/concurrent-merkle-tree/**"
+      - "ci/*-version.sh"
+  push:
+    branches: [master]
+    paths:
+      - "account-compression/**"
+      - "ci/*-version.sh"
```

Sorry, you'll need it here too
```suggestion
      - "account-compression/**"
      - "libraries/concurrent-merkle-tree/**"
      - "ci/*-version.sh"
```